learn.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+   <meta charset="UTF-8">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <script
+      src="https://kit.fontawesome.com/2d323a629b.js"
+      crossorigin="anonymous">
+   </script>
+   <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto&display=swap"
+   rel="stylesheet">
+   <link rel="stylesheet" href="css/tailwind_style.css">
+   <link rel="stylesheet" href="css/style.css">
+   <title>Learn | QC@UCI</title>
+   <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
+
+   <meta name="viewport" content="width=device-width, initial-scale=1">
+
+</head>
+<body>
+   <header class="bg-black text-white pt-2 pb-2 pl-8 pr-8>
+      <div id="left"><img src="image/QC@UCI Logo_small.png" width="125" alt="QC@UCI Logo"></div>
+      <nav>
+         <ul class="header__nav">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="join.html">Join</a></li>
+            <li><a href="learn.html">Learn</a></li>
+            <li><a href="team.html">Team</a></li>
+            <li><a href="event.html">Events</a></li>
+            <li><a href="projects.html">Projects</a></li>
+            <li><a href="contact_us.html">Contact</a></li>
+         </ul>
+      </nav>
+   </header> 
+
+    <p style="margin-bottom:180px;"></p>
+
+   <div class="learn__title">What is Quantum Computing?</div>
+   <p style="margin-bottom:50px;"></p>
+
+   <div class="about-main-container">
+    <div class="about-left-section">
+        <p>
+         Quantum computing is a new paradigm of computing which uses physical quantum phenomena to drastically speed up certain computations. Unlike classical computers that rely on bits, which can be either 0 or 1, to process data, quantum computers use qubits which can be any value between 0 and 1, as well as complex values. This allows for much faster computations on problems such as factoring large numbers, which is a key part of modern encryption.
+
+        </p>
+    </div>
+    <div class="about-right-section">
+        <img src="image/qc.png" alt="A picture of quantum mechanics.">
+    </div>
+    </div>
+
+    <div class="learn__title">Applications</div>
+
+    <div class="applications-container">
+        <div class="app-col-1">
+            <h2>Computer Science</h2>
```

Comment:
I would suggest to break down the "computer science" category as it is quite general. For your example of Shor's algorithm, you can title the excerpt with "Cybersecurity". You can also add another mini section titled "Unstructured Search" and state "Grover's Search Algorithm" and "Quantum Walks Algorithm".